### PR TITLE
Update install-additional-r-packages-on-sql-server.md

### PR DESCRIPTION
**Pull request by monamaki**
- https://github.com/Microsoft/sqlmlutils/tree/master/R/dist goes to an old sqlmlutils, not the releases tab. So, customers may download an old version.
- Updating R.exe -e "install.packages('odbc')" to R.exe -e "install.packages('odbc', type='binary')" (to be consistent with the github readme https://github.com/microsoft/sqlmlutils and customer don't get dependency related errors)

### CODE REVIEW

**Comment by PRMerger16** [comment]
@monamaki : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by monamaki** [comment]
@Aniruddh25 @guitarnana

**Review by guitarnana — APPROVED** [review]

**Comment by ktoliver** [comment]
@guitarnana Thank you for your review. In this repo, the article author or a PM should sign off before a pull request is merged.

@garyericson Could you take a look?

Thanks.

**Review by garyericson — APPROVED** [review]
These changes look good. Thanks for catching this and making the updates.

**Pull request merged**
